biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -533,13 +580,26 @@ real(r8) :: tcwood             ! Temperature response function for wood
                      ! the sapwood pools.
                      ! Units are in (kgN/plant)
                      ! ------------------------------------------------------------------
-                     live_stem_n = EDPftvarcon_inst%allom_agb_frac(currentCohort%pft) * currentCohort%bsw / &
-                           woodcn(currentCohort%pft)
-                     live_croot_n = (1.0_r8-EDPftvarcon_inst%allom_agb_frac(currentCohort%pft)) * currentCohort%bsw  / &
-                           woodcn(currentCohort%pft)
-                     froot_n       = currentCohort%br / frootcn(currentCohort%pft) 
-                     
+
+                     select case(hlm_parteh_model)
+                     case (1)
+                        
```

Comment:
does case 1 have a name? It's hard to remember all the way through which case does which things, so might be good to write the case names where there are these statements.

Reply:
yup, I forgot to add the case names here.  They show up in different parts of the code, but forgot to add them here.
Will do.